src/analyzer/tests/cookies.js

```diff
@@ -107,11 +106,15 @@ export function cookiesTest(
     for (const cookie of allCookies) {
       // Is it a session identifier or an anti-csrf token?
       const sessionId = ["login", "sess"].some((i) =>
-        cookie.key.toLowerCase().includes(i)
+        cookie.key?.toLowerCase().includes(i)
```

Comment:
Nit: If we can reference `cookie.key` in the `filterCookies` function instead of `cookie?.key`, then shouldn't we also be able to omit the `?` here?

Reply:
Unfortunately, ts does not seem to grasp that